application-admintools-default/src/main/resources/templates/securityTemplate.vm

```diff
@@ -17,34 +17,43 @@
-      $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.title'))</h2>
-    <p>$escapetool.xml($services.localization.render('adminTools.dashboard.section.security.description'))</p>
-    <hr/>
-    <ul>
-      <li>
-        $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.activeEncoding')):
-        $security['activeEncoding']
-      </li>
-      <li>
-        $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.configurationEncoding')):
-        $security['configurationEncoding']
-      </li>
-      <li>
-        $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.PWD')): $security['PWD']
-      </li>
-      <li>
-        $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.LANG')): $security['LANG']
-      </li>
-      <li>
-        $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.fileEncoding')):
-        $security['fileEncoding']
-      </li>
-    </ul>
-  </div>
+#includeMacros('Licenses.Code.VelocityMacros')
+#set ($mainReference = $services.model.createDocumentReference('', ['AdminTools', 'Code'], 'ConfigurationClass'))
+#if ($services.licensing.licensor.hasLicensureForEntity($mainReference))
+  #if($security['serverFound'] == 'true')
+    <div class="adminToolsDashboardItem">
+      <h2>$services.icon.renderHTML('lock')
+        $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.title'))</h2>
+      <p>$escapetool.xml($services.localization.render('adminTools.dashboard.section.security.description'))</p>
+      <hr/>
+      <ul>
+        <li>
+          $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.activeEncoding')):
+          $security['activeEncoding']
+        </li>
+        <li>
+          $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.configurationEncoding')):
+          $security['configurationEncoding']
+        </li>
+        <li>
+          $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.PWD')): $security['PWD']
+        </li>
+        <li>
+          $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.LANG')): $security['LANG']
+        </li>
+        <li>
+          $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.fileEncoding')):
+          $security['fileEncoding']
+        </li>
+      </ul>
+    </div>
+  #else
+    #set($warningMessage = $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.error')))
+    #warning($warningMessage)
+  #end
 #else
-  #set($warningMessage = $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.error')))
-  #warning($warningMessage)
+  #includeMacros('Licenses.Code.VelocityMacros')
```

Comment:
Do you need to include it twice? You also include it at the top of the template.

Reply:
Missed that field. I removed the first call for the macro.